### PR TITLE
Feature/localised categories

### PR DESCRIPTION
**Pull request by mozzie2004**

### CODE REVIEW

File: packages/frontend/store/category/actions.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
-import type { Http } from '@vse-bude/shared';
+import type { Http, HttpAcceptLanguage } from '@vse-bude/shared';
 import { getAllCategories, getAllCategoriesSSR } from 'services/category';
 import { CategoryActions } from './action-types';
 
 interface RequestOptions {
   limit?: number;
+  locale?: HttpAcceptLanguage;
```
Comment (Nightlngale) [inline packages/frontend/store/category/actions.ts:8]: You don't need to pass it.

File: packages/backend/src/repositories/category.ts [hunk 2]
```diff
@@ -7,9 +7,11 @@ export class CategoryRepository {
     this._dbClient = prismaClient;
   }
 
-  public getAll(take: number): Promise<Category[]> {
+  public getAll(query: Query): Promise<Category[]> {
+    const { limit = 10 } = query;
```
Comment (Nightlngale) [inline packages/backend/src/repositories/category.ts:11]: Let's just return all the categories

File: packages/backend/src/services/category.ts [hunk 3]
```diff
@@ -7,9 +8,16 @@ export class CategoryService {
...
-  public getAll(query: Query) {
-    const take = query.limit ? +query.limit : 10;
+  public async getAll(req: Request) {
+    try {
+      const result = await this._categoryRepository.getAll(req.query);
 
-    return this._categoryRepository.getAll(take);
+      return result.map((item) => ({
+        ...item,
+        title: req.t(`categories.${item.title}`),
```
Comment (Nightlngale) [inline packages/backend/src/services/category.ts:17]: please return title(or a key which can be used as identifier of FE side for filters, etc.)

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/store/category/actions.ts (hunk 1 above)
Comment (mozzie2004) [reply packages/frontend/store/category/actions.ts:8]: how to get localized data?
should the translation be at the front?

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**